manifests/profile/haproxy.pp

```diff
@@ -19,20 +19,19 @@
     master => $master
   }
 
-  file { '/etc/haproxy/haproxy.cfg':
-    ensure  => 'present',
-    mode    => '0644',
-    content => template('nebula/profile/haproxy/haproxy.cfg.erb'),
-    require => Package['haproxy'],
-    notify  => Service['haproxy'],
-  }
-
-  file { '/etc/default/haproxy':
-    ensure  => 'present',
-    mode    => '0644',
-    content => template('nebula/profile/haproxy/default.erb'),
-    require => Package['haproxy'],
-    notify  => Service['haproxy'],
+  file {
+    default:
+      ensure  => 'present',
+      mode    => '0644',
+      require => Package['haproxy'],
+      notify  => Service['haproxy'],
+    ;
+    '/etc/haproxy/haproxy.cfg':
+      content => template('nebula/profile/haproxy/haproxy.cfg.erb');
+    '/etc/default/haproxy':
+      content => template('nebula/profile/haproxy/default.erb');
+    '/etc/haproxy/errors/hsts400.http':
+      source  => 'puppet:///modules/nebula/haproxy/errors/hsts400.http';
```

Comment:
Was confused for a second about `puppet:///` instead of a template, but I see, it's in `files` instead of `templates`.

Reply:
Yeah, it's full of "^M" line endings, and I'm pretty sure they are important. Not letting erb touch that.
(see: https://github.com/haproxy/haproxy/blob/master/examples/errorfiles/400.http)